src/components/interactions/BaseInteraction.tsx

```diff
@@ -174,15 +176,19 @@ function SingleInteractionInner({
       rawApproveAmount: "0",
     };
   }, [values.fromAmount, values.fromToken, tokenLookup]);
-  const cachedValues = useCachedValue(values)
-  const [lastCalculatedInput, setLastCalculatedInput] = useState<SingleInteractionValues>(values)
+  const cachedValues = useCachedValue(values);
```

Comment:
Since `cachedValues` isn't used anywhere else, the expression `useCachedValue(values)` can be used directly as an argument for `useDebounce`.

`const debouncedValues = useDebounce(useCachedValue(values), 200);`